composer.json

```diff
@@ -18,9 +18,12 @@
   ],
   "require": {
     "php": ">=7.1",
+    "ext-zip": "*",
```

Comment:
All these new dependencies should be optional (if you use Panther as a lib, you may want to use a local ChromeDriver version, or only run this command at build time). I suggest to move these deps in the `require-dev` section and to throw an exception in the command if they are missing.